src/importexport/musicxml/internal/musicxml/importmxmlpass2.cpp

```diff
@@ -2798,8 +2800,14 @@ void MusicXMLParserDirection::direction(const QString& partId,
     //       qPrintable(_wordsText), qPrintable(_rehearsalText), qPrintable(_metroText), _tpoSound);
 
     // create text if any text was found
-
-    if (_wordsText != "" || _rehearsalText != "" || _metroText != "") {
+    if (isLikelyCredit(tick)) {
+        Text* t = Factory::createText(_score->dummy(), TextStyleType::SUBTITLE);
+        t->setLayoutToParentWidth(true);
+        t->setXmlText(_wordsText);
+        auto firstMeasure = _score->measures()->first();
+        VBox* vbox = firstMeasure->isVBox() ? toVBox(firstMeasure) : MusicXMLParserPass1::createAndAddVBoxForCreditWords(_score);
+        vbox->add(t);
+    } else if (_wordsText != "" || _rehearsalText != "" || _metroText != "") {
```

Comment:
Seems to collide with #8678, which , as far as I can tell, got merged into 3.6.2_backend later